sdk/core/Azure.Core/src/Serialization/ModelSerializerFormat.cs

```diff
@@ -11,13 +11,13 @@ namespace Azure.Core.Serialization
     /// </summary>
     public readonly partial struct ModelSerializerFormat : IEquatable<ModelSerializerFormat>
     {
-        internal const string DataValue = "D";
+        internal const string JsonValue = "J";
         internal const string WireValue = "W";
 
         /// <summary>
         /// Specifies the data format where IgnoreReadOnly and IgnoreAdditionalProperties are false.
         /// </summary>
-        public static readonly ModelSerializerFormat Data = new ModelSerializerFormat(DataValue);
+        public static readonly ModelSerializerFormat Json = new ModelSerializerFormat(JsonValue);
```

Comment:
Will the doc string here make sense to most users of this API? Reading it without additional context, I'm not exactly sure what it means.

Reply:
We are address doc comments in a different PR, but yes these are not final